.github/workflows/eleventy_build.yml

```diff
@@ -0,0 +1,27 @@
+name: Eleventy Build
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Shouldn't this be `main` only?

Reply:
@zcorpan I left the setting for all branches to naively handle previews. I can put it on main. Are previews even necessary?

Reply:
Having branches replace what is deployed on the main URL seems worse than not having previews at all. We can preview locally for now.